client/my-sites/checkout/src/test/composite-checkout-variant-picker-dropdown.tsx

```diff
@@ -162,6 +163,62 @@ describe( 'CheckoutMain with a variant picker', () => {
 		}
 	);
 
+	it.each( [
+		{ activePlan: 'none', cartPlan: 'monthly', expectedVariant: 'yearly' },
+		{ activePlan: 'none', cartPlan: 'monthly', expectedVariant: 'two-year' },
+		{ activePlan: 'none', cartPlan: 'yearly', expectedVariant: 'monthly' },
+		{ activePlan: 'none', cartPlan: 'yearly', expectedVariant: 'two-year' },
+		{ activePlan: 'none', cartPlan: 'two-year', expectedVariant: 'monthly' },
+	] )(
+		'renders the $expectedVariant variant with a discount percentage for a Jetpack $cartPlan plan when the current plan is $activePlan',
+		async ( { activePlan, cartPlan, expectedVariant } ) => {
+			( getPlansBySiteId as jest.Mock ).mockImplementation( () => ( {
+				data: getActivePersonalPlanDataForType( activePlan ),
```

Comment:
This looks like it's getting some data for test Personal plans, not sure if this needs to be Jetpack specific too?

Reply:
Right now I am not testing any cases with `activePlan` here so it looks like we don't need to use that function in this case. I'll go ahead and remove that part